src/components/value/index.tsx

```diff
@@ -263,6 +307,31 @@ export function Value({
           placeholder={dataType === "datetime" ? "yyyy-mm-ddThh:mm" : undefined}
         />
       )}
+      {edit && dataset && thing && inputType === "datetime-workaround" && (
+        <>
+          <input
+            type="date"
+            aria-label="Date"
+            step={inputStep}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...inputProps}
+            onChange={(e) => {
+              setDate(e.target.value);
+            }}
+            onBlur={(e) => autosave && saveHandler(e)}
+            value={date}
+            pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}"
+            placeholder="yyyy-mm-dd"
+          />
+          <input
```

Comment:
Probably useful to add a `pattern` and a `placeholder` to the time input too.

Reply:
Added a `pattern`, but `placeholder` does not work with time input.

Reply:
I suppose it's mostly for browsers that do not support it, and thus fall back to `text`? Either way I don't think those are very common, so a minor issue really.